src/main/java/com/cactus/systems/GroceryListSystem.java

```diff
@@ -94,5 +95,40 @@ public boolean exitGroceryList(){
         }
     }
 
+    /***
+     * Add grocery items to the current grocery list
+     *
+     * @param items  list of grocery items
+     * @param userid id for the current user
+     * @return true if items were added, false otherwise
+     */
+    public boolean addGroceryItems(List<String> items, long userid){
+        Response setGroceryItemResponse =
+                this.groceryListManager.setGroceryItems(items, this.currentGroceryListId, userid);
```

Comment:
Note that `setGroceryItems` simply overwrites the existing grocery items with whatever you feed it. This means that should you want to add items one at a time, that's logic that needs to happen at a level higher than the use cases.
This was a design choice on my part because I wasn't sure if we wanted to do a "save file" type of operation, or an add items one by one implementation like the UI currently supports. Either way, the controller should decide what kind of operation it wants to support, and use the `setGroceryItems` method accordingly.